ManualText.txt

```diff
@@ -308,21 +325,23 @@ o	NOTE: If there is NO response, immediately repeat, "[Officer Number], I have y
 •	After you have been asked to standby, if you receive no "go ahead" after 5 minutes, you should radio the officer to check status.
 •	Never simply go ahead reading the listing until the officer has given you a “go ahead” as it can be distracting to the officer if they are at the car with the operator, or incite argument if overheard by the operator.
 
-3.7	Openers
-Student openers come in six days a week during normal school operation. They have a radio and call in after they open each building. You will log each building after they open it the same way you would if an officer called in. Include times when replying to openers on the radio.
+3.7	Openers, Closers, and PEAs
+Student openers come in six days a week during normal school operation. Put them on duty in Report Exec. when they begin their shift. They have a radio and call in after they open each building. You will log each building after they open it as a officer status the same way you would if an officer called in. Include times when replying to openers on the radio.
```

Comment:
We should add something about them calling in at the start and end of an unlocking